dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -450,9 +453,9 @@ export const Card = ({
 -	 * Media cards have contrasting background colours. We add additional
 	 * padding to these cards to keep the text readable.
 -	 */
-	const hasBackgroundColour = isMediaCard(format);
+	const isMediaCard = isAMediaCard(format);
```

Comment:
This is a better name for this var 👍  I lazily left it alone as it clashed with the method name.